### PR TITLE
Do some Attachment stuffs

### PR DESCRIPTION
**Pull request by cfpwastaken**
- Added new Attachment types
- Added new extensions

WIP:
- Custom Video & Audio Player

Known bugs that **need** to be fixed before merge:
- If there are video attachments, it somehow will scroll up. Dunno why tho

### CODE REVIEW

File: util/div-creator.js [hunk 1]
```diff
@@ -151,16 +151,47 @@ function createAttachmentDiv(a) {
 			img.addEventListener("click", handler);
 			div.appendChild(img);
 		}
+	} else if(isVideoExt(a.name)) {
+		if(a.width > 1920 || a.height > 1080) {
+			var text = document.createElement("text");
+			text.className = "embed-errtxt attachment";
+			text.innerHTML = "Refused to load video due to it being too big.";
```
Comment (j0code) [inline util/div-creator.js:158]: needs `<br>Click to view` and click listener to open the video in a new tab (-> popup, links are spooky)

File: util/div-creator.js [hunk 2]
```diff
@@ -151,16 +151,47 @@ function createAttachmentDiv(a) {
...
+		div.appendChild(audio);
 	} else {
+		// var text = document.createElement("text");
+		// text.className = "embed-errtxt attachment";
+		// text.innerHTML = "Unknown Attachment Type: " + a.name;
+		// div.appendChild(text);
 		var text = document.createElement("text");
 		text.className = "embed-errtxt attachment";
-		text.innerHTML = "Unknown Attachment Type: " + a.name;
+		text.innerText = "Unknown Attachment Type: " + a.name + " - " + a.url;
```
Comment (j0code) [inline util/div-creator.js:180]:
```suggestion
		text.innerText = "Unknown Attachment Type: " + a.name + "\n" + a.url;
```

File: util/div-creator.js [hunk 3]
```diff
@@ -151,16 +151,47 @@ function createAttachmentDiv(a) {
...
 		div.appendChild(text);
 	}
 	//console.log(a);
 	return div;
 }
 
 function isImageExt(f) {
-	return f.endsWith(".png") || f.endsWith(".jpg") || f.endsWith(".jpeg") || f.endsWith(".webp");
+	return f.endsWith(".png") || f.endsWith(".jpg") || f.endsWith(".jpeg") || f.endsWith(".webp") || f.endsWith(".svg") || f.endsWith(".gif") || f.endsWith(".tif") || f.endsWith(".tiff") || f.endsWith(".eps") || f.endsWith(".bmp");
```
Comment (j0code) [inline util/div-creator.js:188]: .eps ?

**Review by j0code — COMMENTED** [review]
nice

File: util/div-creator.js (hunk 2 above)
Comment (cfpwastaken) [reply util/div-creator.js:180]: Ooop sry

File: util/div-creator.js (hunk 3 above)
Comment (cfpwastaken) [reply util/div-creator.js:188]: Looks like .eps is not just some weird video format

File: util/div-creator.js (hunk 1 above)
Comment (cfpwastaken) [reply util/div-creator.js:158]: Ok

Comment (cfpwastaken) [reply util/div-creator.js:158]: Doing it later

**Comment by cfpwastaken** [comment]
I guess I can merge this now?

**Comment by cfpwastaken** [comment]
Please don't kill me if i wasn't supposed to do this @j0code

**Pull request merged**